### DISCUSSION TITLE
Several questions

### DISCUSSION THREAD

**Discussion by vinniec2** [ruanformigoni/gameimage#84] [Q&A]
I saw your project some time ago and now I wanted to try it.

I chose a game for dos and so I aimed for retroarch, but I couldn't get it to work, these are the problems I encountered till now:
1) it seems to me that the interface assumes that the rom is a single file, whereas my game was in a folder. I know it's maybe possible to create a zip or maybe an iso, though so far I've only tried by hand inserting the folder inside the ./rom subdir but I couldn't get it to work. Is there by any chance a tutorial?
2) now I'm trying with wine, also because wine should also support dos games and it seems to me that seeing your video, I can select a directory (I'm not sure if I remember correctly but I'll try).
3) I had looked for an issue on the tracker and a person talking about fltk (which I like) asked you to enlarge the font.
Yes, I ask that too, in my machine it's necessary, the font is so small that you can't read it even if you zoom in because there aren't enough pixels to render the letters!
Maybe I could raise the dpi of the whole desktop but it would be helpful to be able to choose a zoom factor of the interface!
4) it might be a good idea to create a cache of downloaded files from your program? pretend I wanted to create10 gameimage, currently I have to download wine 10 times :D

otherwise, great project, I've always liked appimages but the idea you had to make a container without however forcing to install anything like flatpack (which instead together with snap I don't like) Is really valid.
...Sorry another stupid question, is your container based on any distro?

**Comment by vinniec2** [comment]
nevermind, I couldn't do it even with wine, when it asks me to install/test the files, I don't know how to pass the directory to it. I always tried to manually put it in ./build/game/rom but still couldn't get through :P

**Comment by ruanformigoni** [comment]
Hi @vinniec2, thanks a lot for using gameimage and reporting these issues! 

## Multiple files on the rom screen

To select multiple files on the room screen, you can hold CTRL and click on the ones you want to include. After they are included, you can selected the default one for retroarch to open with the blue checkmark button. 

## Download cache location

The build folder for gameimage, selected in the initial screen is the default cache folder. If you select it again afterwards, it will find the already downloaded files and use them instead. This is nice for the user to know where the cached files are, since they are considerably large.

## Use wine with a pre-existing directory

If you want to use a pre-existing directory with wine, instead of using an installer, you can:
- Click on the folder icon on any executable on the "select executable screen"
- This will take you to inside the C: drive
- Go you a few directories until you are in the `drive_c` folder.
- Copy your directory to this folder
- Go back to the gameimage wizard
- Click on `Prev` to go back to the  `Configure Wine` screen
- Click on `Next` to go to the `Install/Test Application(s)` screen
- Now you executables should be appearing there

## Other

The font size is indeed small, I'll try to improved that.

> is your container based on any distro?

Yes, it is arch linux.

If there are any more questions or issues, feel free to ask. Thanks again!
